src/App.tsx

```diff
@@ -1,24 +1,120 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
+import { deleteTodo, getTodos } from './api/todos';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { TodoItem } from './components/TodoItem';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Options } from './types/Options';
+import { getFilteredTodos } from './utils/filterTodos';
+import { ErrorMessages } from './types/ErrorMessages';
 
-const USER_ID = 0;
+const USER_ID = 6345;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<string>(
+    ErrorMessages.NOERROR,
+  );
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [filterType, setFilterType] = useState(Options.ALL);
+  const [loadingTodoIds, setLoadingTodoIds] = useState([0]);
```

Comment:
`loadingTodoIds` - says about boolean not array of ids, let us rename it

Reply:
I save at `loadingTodoIds` ids of loading `todos` because sometimes we need a couple of todos to have a `loader` at a time.
If I got it wrong, tell me, please.